compiler/rustc_feature/src/active.rs

```diff
@@ -576,6 +576,12 @@ declare_features! (
     /// Allows using and casting function pointers in a `const fn`.
     (active, const_fn_fn_ptr_basics, "1.48.0", Some(57563), None),
 
+    /// Allows trait bounds in `const fn`.
+    (active, const_fn_trait_bound, "1.53.0", Some(57563), None),
+
+    /// Allows unsizing coercions in `const fn`.
+    (active, const_fn_unsize, "1.53.0", Some(64992), None),
```

Comment:
I have no idea what this version number is for -- this macro syntax is not very self-explaining...

Reply:
I think it is edition stuff, ignore it